descriptors/ModuleDescriptor-template.json

```diff
@@ -488,13 +488,13 @@
       ]
     },
     {
-      "id": "instance-bulk",
+      "id": "record-bulk",
```

Comment:
I am not sure if renaming an interface was done previously in FOLIO, but we need to increment the version as well

Reply:
Also verify if this interface or API is used anywhere else in FOLIO other than the ui-inventory, we need to make those changes in that modules as well

Reply:
UI-inventory, data-export and perf-testing are using this interface

Reply:
Thanks! Let's add this list as TODO and when PRs are ready tag them here

Reply:
updated